pom.xml

```diff
@@ -46,6 +46,12 @@
 			<artifactId>spring-cloud-starter-openfeign</artifactId>
 		</dependency>
 
+		<dependency>
+			<groupId>io.github.openfeign</groupId>
+			<artifactId>feign-httpclient</artifactId>
+			<version>12.5</version>
```

Comment:
Czy zweryfikowałeś wersje czy jest kompatybilna z Klientem Feignowym dodawanym przez spring-cloud-starter-openfeign?

Reply:
tak, dziala to poprawnie